src/libxively/xi_globals.h

```diff
@@ -33,6 +33,9 @@ typedef struct
     char* str_account_id;
     char* str_device_unique_id;
     xi_backoff_status_t backoff_status;
+
+    char** updateable_files;
+    uint16_t updateable_files_count;
```

Comment:
hm, unfortunately you are right!
ok, now there is a new API for file updates:
```
extern xi_state_t
xi_initialize_add_updateable_files( const char* filenames[], uint16_t count );
```

I will remove this and add a new with context:

```
extern xi_state_t
xi_set_updateable_files( xi_context_handle_t context_handle, const char* filenames[], uint16_t count );
```

Is is ok?